install_config/configuring_sdn.adoc

```diff
@@ -212,37 +212,41 @@ $ oc get netnamespace
 ----
 
 [[migrating-between-sdn-plugins-networkpolicy]]
-=== Migrating from the ovs-multitenant Plugin to ovs-networkpolicy
+=== Migrating from ovs-multitenant to ovs-networkpolicy
 
-Before migrating from *ovs-multitenant* to *ovs-networkpolicy*, you must ensure that every
-namespace has a unique `NetID`; that means that if you have previously
+Before migrating from the *ovs-multitenant* plugin to the *ovs-networkpolicy*
+plugin, ensure that every namespace has a unique `NetID`. This means that if you
+have previously
 xref:../admin_guide/managing_networking.adoc#joining-project-networks[joined projects
 together] or
 xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made projects
-global], you will need to undo that before switching to the *ovs-networkpolicy* plugin, or
-NetworkPolicy objects may not function correctly.
+global], you will need to undo that before switching to the *ovs-networkpolicy* plugin,
+or the NetworkPolicy objects may not function correctly.
 
-A helper script is available that fixes `NetID`s, creates NetworkPolicy objects to isolate
-previously-isolated namespaces, and enabled connections between previously-joined
-namespaces.
+A helper script is available that fixes `NetID's`, creates NetworkPolicy objects
+to isolate previously-isolated namespaces, and enables connections between
+previously-joined namespaces.
 
-. Download https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/migrate-network-policy.sh
+Use the following steps to migrate to the *ovs-networkpolicy*
+plugin, by using this helper script, while still running the *ovs-multitenant* plugin:
+
+. Download the script from link:https://raw.githubusercontent.com/openshift/origin/master/contrib/migration/migrate-network-policy.sh[] and add the exexcution file permission:
 +
```

Comment:
Did you test this link? I cannot get this or the un-migrate link to do anything.